finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,11 @@
+var frogs = new Array(10); 
+
+for (var i = 10; i > 0; i--) {
+  if ([i] > 2) {
+    console.log([i] + " speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool, then there were ", + [i] - 1, " speckled frogs.");
+} else if ([i] == 2) {
+    console.log([i] + " speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool, then there was 1 speckled frog.");
+} else{
```

Comment:
Try to make your `else` statement your default- the case that will be run the most amount of times. I may switch this with your if statement, changing the logic as well.

Reply:
Updated and made note on which else statement should be the default going forward.